Formula/ocmtoc.rb

```diff
@@ -0,0 +1,37 @@
+class Ocmtoc < Formula
+  desc "Mach-O to PE/COFF binary converter"
+  homepage "https://github.com/acidanthera/ocmtoc"
+  url "https://github.com/acidanthera/ocmtoc/archive/refs/tags/1.0.3.tar.gz"
+  sha256 "9954194f28823e4b1774d2029a1d043e63b99ff31900bff2841973a63f9e916f"
+  license "APSL-2.0"
+
+  depends_on :macos
+  conflicts_with "mtoc", because: "both install `mtoc` binaries"
```

Comment:
Add this line in the separate PR so CI will run.

Reply:
Finally understood you, done.